redaxo/src/addons/be_style/scss/_mime.scss

```diff
@@ -18,10 +19,13 @@
     text-transform: uppercase;
 }
 .rex-mime-default {
+    @extend .fa-regular;
+    @extend .fa-file;    
 }
 .rex-mime-doc, 
 .rex-mime-docx {
-    @extend .fa-file-word-o;
+    @extend .fa-regular;
```

Comment:
Weshalb benötigst du diese Zeile? .fa-regular notierst du bereits in Zeile 3.
Gilt auch für alle weiteren Mime-Icons.

Reply:
Hallo Thomas, es gibt sowohl regular als auch solid. Ich gehe erst einmal davon aus, dass es beide braucht. Eventuell müssen wir das oben löschen. Doch dann funktioniert vermutlich die default-Version nicht. Wie würdet Ihr da Font Awesome verstehen?

Reply:
> s gibt sowohl regular als auch solid

das ist mir bewusst. Wir zeichnen die mime-Icons so aus
`class="rex-mime rex-mime-pdf"`

Wenn du jetzt bei `.rex-mime` bereits Regular hinterlegst, dann benötigt `.rex-mime-pdf` diese Zeile nicht mehr.

Reply:
Ah, das könnte die Ursache für den Fehler oben sein. Heißt in meinem Verständnis: Wir müssen bei der Zuordnung zu den Klassen das rex-mime (die allgemeine Version) weglassen. Dann müsste es die Icons richtig ausgeben. Offensichtlich beißt sich das nämlich mit der neuen Denke von Font Awesome.

Wo müsste ich das anpassen?